tests/e2e/lib/k8s_common_helpers.go

```diff
@@ -209,7 +209,7 @@ func GetFirstPodByLabel(clientset *kubernetes.Clientset, namespace string, label
 func SavePodLogs(clientset *kubernetes.Clientset, namespace, dir string) error {
 	podList, err := clientset.CoreV1().Pods(namespace).List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		return nil
+		return err
```

Comment:
I would further refactor here

there are other 3 `if err != nil` statements, that do not return an error

(This function only runs if a test fails, so changing its behavior seems harmless to me)

Reply:
Added returns